mpas_analysis/test/test_mpas_xarray.py

```diff
@@ -38,7 +40,8 @@ def test_subset_variables(self):
                                                 calendar=calendar,
                                                 timeVariableName=timestr)
         ds = mpas_xarray.subset_variables(ds, variableList)
-        self.assertEqual(sorted(ds.data_vars.keys()), sorted(variableList))
+        dsVarList = list(ds.data_vars.keys()) + list(ds.coords.keys())
+        assert(numpy.all([var in dsVarList for var in variableList]))
```

Comment:
@xylar, why not an `self.assertEqual` instead of requiring `numpy` here (and other places)?

Reply:
@pwolfram, the issue is that `variableList` is a *subset* of the sum of `data_vars` and `coords`.  There are other coords such as `Time`.  So `self.assertEqual` is not correct.

Reply:
One could change this to:
```
self.assertArrayEqual([var in dsVarList for var in variableList], True)
```
but i don't personally find this more intuitive or preferable to `numpy.all()`